frameworks/hdfs/universe/marathon.json.mustache

```diff
@@ -29,6 +29,7 @@
     "FRAMEWORK_NAME": "{{service.name}}",
     "FRAMEWORK_LOG_LEVEL": "{{service.log_level}}",
     "HDFS_VERSION": "hadoop-2.6.0-cdh5.11.0",
+    "TASKCFG_ALL_HDFS_VERSION": "hadoop-2.6.0-cdh5.11.0",
```

Comment:
In which cases do we need both the regular and `TASKCFG_ALL_*` variants of an environment variable?

Reply:
The env vars don't need to be prefixed if they're not sent to any tasks essentially. So `HDFS_VERSION` will only be rendered in the scheduler's environment, whereas `TASKCFG_ALL_HDFS_VERSION` will get pushed through to all the tasks' environments (say to render a template, like `hdfs-site.xml`)

In this case, `TASKCFG_ALL_HDFS_VERSION` seems like doesn't actually need to be here as it's not used in any task environments, so will remove.